README.md

```diff
@@ -0,0 +1,104 @@
+
+- Set the `NOMAD_ADDR` to the IP-address or hostname of a Nomad cluster that is routable for GitHub Actions Runners.
+- Set the `NOMAD_TOKEN` to a token with appropriate permissions to carry out Pack-specific operations on a Nomad cluster.
+
+> **Warning**
+> Running services such as Nomad on a publicly accessible port without authentication is a decidedly bad idea.
+>
+> Consult with your security team to define an access policy that meets your organization's security demands.
+
+<small>GitHub Actions run on a publicly-known list of [IP addresses](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#ip-addresses).
+This data may be retrieved through [HashiCorp Terraform](https://terraform.io/), using the [ip_ranges data source](https://registry.terraform.io/providers/integrations/github/latest/docs/data-sources/ip_ranges), allowing you to make IP-address _one_ of the security considerations.</small>
+
+2.) Create a GitHub Actions Workflow file (e.g.: `.github/workflows/nomad-pack.yml):
+
+```yaml
+name: nomad-pack
+
+on:
+  - push
+
+jobs:
+  setup-nomad-pack:
+    runs-on: ubuntu-latest
+    name: Run Nomad Pack
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup `nomad-pack`
+        # TODO: define `v1`
+        uses: hashicorp/setup-nomad-pack@v1
+        id: setup
+        with:
+          version: "0.0.1-techpreview2"
+
+      - name: Run `nomad-pack info` for `simple_service`
+        run: "nomad-pack info ./packs/simple_service"
+
+      - name: Run `nomad-pack run` for `simple_service`
+        run: "nomad-pack run ./packs/simple_service"
+        env:
+          NOMAD_ADDR: "${{ secrets.NOMAD_ADDR }}"
+          NOMAD_TOKEN: "${{ secrets.NOMAD_TOKEN }}"
+        continue-on-error: true
+```
+
+In the above example, the following definitions have been set.
+
+- The event trigger has been set to `push`. For a complete list, see [Events that trigger workflows](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows).
+- The origin of this GitHub Action has been set as `ksatirli/setup-nomad-pack@0.1.3`. For newer versions, see the [Releases](https://github.com/ksatirli/setup-nomad-pack/releases).
+- The version of `nomad-pack` to set up has been set as `0.0.1-techpreview2`. For a complete list, see [releases.hashicorp.com](https://releases.hashicorp.com/nomad-pack/).
+- The pack to deploy has been set as `./packs/simple_service`
+
+These definitions may require updating to suit your deployment.
+
+## Inputs
+
+This section contains a list of all inputs that may be set for this Action.
+
+- `version` - (required) The version of `nomad-pack` to install. Supports [semver](https://www.npmjs.com/package/semver) versioning. Defaults to `latest`.
+
+## Outputs
+
+This section contains a list of all outputs that can be consumed from this Action.
```

Comment:
This is I think where you'd want to define/trap the metadata outputs for what the pack rendered ya?

Reply:
Ah, so! `outputs` here are the generic outputs of the GitHub Action.

You _can_ consume the return values of a (previous) step though, so I'll add some links to docs for that. Describing that in-depth is more involved, as it is somewhat specific.